### ISSUE TITLE
how to use gui in the AidLearning?

### ISSUE THREAD

**Issue by paint2android** [aidlearning/AidLearning-FrameWork#9]
how to use gui in the AidLearning?how to custom the gui?

**Comment by aidlearning** [comment]
you can see this 👍 https://github.com/aidlearning/AidLearning-FrameWork/issues/5#issuecomment-489677667

**Comment by qfmail** [comment]
templates folders can customize your ui

**Comment by aidlearning** [comment]
yes! you can use mkdir templates folder to customize you gui. AidLearning usr the html define the caption of dialog ,define the input ,info area....
